File: Python/2-locating-elements/5-By-Link-Text.py
from selenium import webdriver

driver = webdriver.Chrome()

try:
    driver.get('https://selenium.dev/documentation/en/')

    '''
    Fetch the below DOM node
    <a href="//www.w3.org/TR/webdriver/" class="highlight">W3C WebDriver specification</a>
    '''
    my_link = driver.find_element_by_link_text("W3C WebDriver specification")

    # Access the attributes value
    print(my_link.get_attribute('href'))
    my_link.click()
finally:
    driver.quit()

File: Javascript/2-locating-elements/5-By-Link-Text.js
const {Builder, By} = require('selenium-webdriver');

(async function(){
    const driver = new Builder().forBrowser('chrome').build();

    try {
        (await driver).get("https://selenium.dev/documentation/en/")

        /**
         * Fetch the below DOM node
         * <a href="//www.w3.org/TR/webdriver/" class="highlight">W3C WebDriver specification</a>
         */
        let my_link = (await driver).findElement(By.linkText("W3C WebDriver specification"));
        let my_href = (await my_link).getAttribute('href');
        console.info(my_href);

        (await my_link).click();
       

    } finally {
        (await driver).quit();
    }

})();